homeassistant/components/demo/device_tracker.py

```diff
@@ -21,7 +25,7 @@ def random_see(dev_id, name):
             battery=random.randrange(10, 90),
         )
 
-    def observe(call=None):
+    def observe(call: ServiceCall | None = None) -> None:
```

Comment:
`observe` is called here without parameters:
https://github.com/home-assistant/core/blob/a8fe9f2eff70d83fd347e3d2fa94d9730ab77eaf/homeassistant/components/demo/device_tracker.py#L29